### ISSUE TITLE
Unwanted grade conversion in "Notable Ascents"

### ISSUE THREAD

**Issue by quaestor** [theCrag/website#3880]
**What happened?**
In my "notable ascents" on the profile page, a route that is graded "E3 5c" (but with a crag grading context of "FR") is converted to the grade "7" (but without the rounded corners) without any apparent reason. My preferred Trad context is set to "Let system decide".

Example url(s) to reproduce the problem:
https://www.thecrag.com/climber/quaestor

Ideally also provide screenshots:
<img width="618" alt="Screenshot 2021-03-24 at 12 35 28" src="https://user-images.githubusercontent.com/54371/112304420-c0f57d00-8c9d-11eb-909b-2fcaaa54f74f.png">
<img width="713" alt="Screenshot 2021-03-24 at 12 35 36" src="https://user-images.githubusercontent.com/54371/112304436-c652c780-8c9d-11eb-9915-68a9c157655a.png">

**What you expected:**
Just display the original grade of "E3 5c".

**Comment by quaestor** [comment]
Bump. Why isn't there a setting that allows to display that list as it used to be before the introduction of grade conversions? As in: just display the native route grade?

**Comment by scd** [comment]
The reason why it is like this is complex user requirements interacting with either a miss understood or missed use case. The profile page is scheduled for rework.

Just to get some requirements clear on what you expect I have a few followup questions. 

A profile is for the benefit of other climbers. If you see somebody else's profile do you want to be able to see notable ascents in a grading system you understand?

If you climb overseas do you want to see notable ascents converted to your local grading system?

Do you think this is just a bug? @quaestor you mentioned that your trad grading system is set to 'Let system decide' so this could be interpretted as a bug? Or do you expect that this should be irrelevant for notable ascents and that they should always be shown with native grade regardless of any user settings or conversions.

**Comment by quaestor** [comment]
> A profile is for the benefit of other climbers.

Good point. However, since there is no "private" *statistics* page, most people probably look more often at their own profiles than at other people's. I know I do (without wanting to sound narcissistic ;-)). Especially the grade pyramids and the CPR charts are useful tools to capture my own progress, not someone else's.

> If you see somebody else's profile do you want to be able to see notable ascents in a grading system you understand?

Actually, I don't. But I agree that I may be in the minority here. The system in which a route is graded often also tells you something about the style it is climbed in, and maybe even a little bit about ethics. Also, I often don't fully agree with the conversion tables. Getting used to the different systems is one of the joys of getting to know different climbing areas. It's not about the numbers for me.

> If you climb overseas do you want to see notable ascents converted to your local grading system?

No, for the reasons described above. If I climbed an E4 6a in Britain (which is technically overseas for me ;-)), I don't want it to show up as an UIAA 8 in my statistics, because the British grade tells so much more. Same goes for the bold Elbe sandstones routes. Converting the grades to UIAA takes away a lot of the information.

> Do you think this is just a bug? @quaestor you mentioned that your trad grading system is set to 'Let system decide' so this could be interpretted as a bug? Or do you expect that this should be irrelevant for notable ascents and that they should always be shown with native grade regardless of any user settings or conversions.

I think what can be considered as a bug is that the conversion is invisible. At the very least, there should be rounded corners and maybe display the original grade on hovering over the grade.

**Comment by scd** [comment]
It is good to get your feedback. We will of course get feedback from multiple climbers. 

One of the issues with the Profile page is that it performs two functions - a genuine profile for others to see and self assessment statistics.

**Comment by DaneEvans** [comment]
Put me down as another one that would rather see original grades, at least
on my own page.

I can't count the number of times I've gone 'a 25? What was that ascent?
Oh, it's a Thai 7a.  I remember it now'


I can see the arguments for a universal display, but having one or the
other on mouse over sounds ideal.  I'd probably say the converted one on
mouseover, and keep the table in original grades.

**Comment by Dane-2pi** [comment]
I've once again come across this one, and it's frustrated me again. 
I then went digging through the settings to see if I could change it back to local context. 

If we do this, then we need to have a Thai system as well as the french that takes 2-3 grades off them

**Comment by killakalle** [comment]
> A profile is for the benefit of other climbers. If you see somebody else's profile do you want to be able to see notable ascents in a grading system you understand?

@scd Whenever I see climber profiles with unfamiliar grades, e.g. I'm not familiar with Australian, I wish the profile page had the "Convert" button that is already present for crags and routes.

**Comment by quaestor** [comment]
@killakalle You're raising an interesting point! If I visit the profile of a climber from the US, all the grades are in YDS, even any notable ascents the climber might have in my own local grading system (UIAA), or maybe even more confusing, a third grading system, like French sport. I'm once again voting for a return to original route grade display. Plus a "convert" button to show everything in *my own* system, for those who need it.
